configuration-reference/functions/distinctcountrawhllplusmv.md

```diff
@@ -0,0 +1,39 @@
+---
+description: >-
+  This section contains reference documentation for the DISTINCTCOUNTRAWHLLPLUSMV
+  function.
+---
+
+# DISTINCTCOUNTRAWHLLPLUSMV
+
+Returns HLLPlus response serialized as string. The serialized HLLPlus can be converted back into an HLLPlus and then aggregated with other HLLPluses. A common use case may be to merge HLLPlus responses from different Pinot tables, or to allow aggregation after client-side batching.
```

Comment:
to be sure, is there any distinction to make between this description and the description for DISTINCTCOUNTRAWHLLPLUS?

Reply:
yes. there is not much difference between the DISTINCTCOUNTRAWHLLPLUS and DISTINCTCOUNTRAWHLLPLUSMV.